src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -65,6 +71,16 @@ Registry.as<IQuickAccessRegistry>(Extensions.Quickaccess).registerQuickAccessPro
 	helpEntries: [{ description: localize('manageExtensionsHelp', "Manage Extensions"), needsEditor: false }]
 });
 
+// Explorer
+MenuRegistry.appendMenuItem(MenuId.ExplorerContext, {
+	group: 'navigation',
```

Comment:
Nice and clean registration.
I would just play with the order and group. This definetly should not be at the top of the navigation group as it is now - it is shown too high in the context menu.
Feels to me like it should be a seperate group, similar to Find in Folder